src/theme/default-theme.js

```diff
@@ -23,10 +23,5 @@ export default {
     text: '44px',
     monospace: '20px'
   },
-  space: {
-    headerMargin: '16px 16px 24px',
-    textMargin: '16px',
-    listMargin: '16px 0',
-    slidePadding: '2em'
-  }
+  space: [16, 24, 32]
```

Comment:
Instead of fixed values in a CSS string use consistent spacing as part of a scale that can be changed by the user.